sda-download/internal/config/config.go

```diff
@@ -163,7 +172,7 @@ func NewConfig() (*Map, error) {
 		}
 	}
 	requiredConfVars := []string{
-		"db.host", "db.user", "db.password", "db.database", "c4gh.filepath", "oidc.configuration.url",
+		"db.host", "db.user", "db.password", "db.database", "c4gh.filepath", "c4gh.passphrase", "oidc.configuration.url",
```

Comment:
Is this intentional or is not rebased and an effect of that?

Reply:
It was removed from sda-download due to the difference in handling password protected keys between our infra and the rust implementation. Since the re-encryption service allows for having different keys, we are putting back the password as required variable.